src/MerchantCenter/PolicyComplianceCheck.php

```diff
@@ -0,0 +1,99 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Infrastructure\Service;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\WC;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class PolicyComplianceCheck
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter
+ *
+ * @since x.x.x
+ */
+class PolicyComplianceCheck implements Service {
+	/**
+	 * The WC proxy object.
+	 *
+	 * @var wc
+	 */
+	protected $wc;
+
+	/**
+	 * BaseController constructor.
+	 *
+	 * @param WC $wc
+	 */
+	public function __construct( WC $wc ) {
+		$this->wc = $wc;
+	}
+
+	/**
+	 * Get the allowed countries for the controller.
+	 *
+	 * @return array
+	 */
+	public function get_allowed_countries(): array {
+		return [ 'allowed_countries' => $this->wc->get_allowed_countries() ];
```

Comment:
Is it necessary to send the full list of allowed countries? Or can we check if the store's country is in the allowed countries list?

Reply:
To check the store's country sounds more efficient. By that way, we might need add the store's country in the request. Could we update this later to improve after we done the UI?